board/drivers/can.h

```diff
@@ -387,16 +463,21 @@ void CAN3_RX0_IRQHandler(void) { can_rx(2); }
 void CAN3_SCE_IRQHandler(void) { can_sce(CAN3); }
 
 void can_send(CAN_FIFOMailBox_TypeDef *to_push, uint8_t bus_number) {
+  bool res;
   if (safety_tx_hook(to_push) != 0) {
     if (bus_number < BUS_MAX) {
       // add CAN packet to send queue
       // bus number isn't passed through
       to_push->RDTR &= 0xF;
       if ((bus_number == 3U) && (can_num_lookup[3] == 0xFFU)) {
         // TODO: why uint8 bro? only int8?
-        gmlan_send_errs += !bitbang_gmlan(to_push);
+        res = bitbang_gmlan(to_push);
```

Comment:
What was wrong with += before?

Reply:
Bool added to a int. Misra does not like it. It's already fixed in master, just need to resolve conflicts.